### ISSUE TITLE
Warning window when camera temp spikes or increases

### ISSUE THREAD

**Issue by PatrinaPellett** [oxfordni/support#4]
Occasionally the coolant for the camera needs to be added. When it runs low, obviously this affects the temperature of the camera. Can you please implement a warning/popup window or some other indication when the camera temp has increased? It would be great if we could also suggest that should check the coolant. Or if there is some other problem to contact ONI support.

**Comment by zmikulski** [comment]
Yes, please! Let that horrible sound be your guiding angel to a speedy implementation. 
https://photos.app.goo.gl/VRrlaKCcWhBOVPYs1

A *smart* self-diagnosis system will be the **first** on microscopy hardware market. Given that you have just opened the doors to almost real-time reporting feature gives you the tools to make it the reality. 

My brother (ask Mike about the stuff he can conjure from simple ingredients) told me once that in his opinion all engineers should spend obligatory two weeks with the thing she or he designed. He is an engineer. I trust him. His still is legendary.

Systems in the field will encounter a vast array of issues. Overheating like this is a no-go for a decently designed product. If you have an X-box controller, you are obliged to have a **kick-ass** onboard diagnostics.  

**Any reaction to given feedback is satisfying**. It would be cool to see that someone has read this. Of course, the solution might be a few stone throws away, but the act of communication is building the positive conjunction with the client. Your client is naturally your ally. If you respond to them, they will tell you about all the problems they encountered, so talk to them and listen with open ears.

**Comment by ollie1400** [comment]
Hi zmikulski,

Thanks for your comment, apologies for the delay in response.  I have been thinking about the best response to an overheating condition; to be safe the microscope should shut itself down if it detects an overheating event (if temperature rises about 30 degrees for example). In the case where the microscope cannot sufficiently cool itself I would suggest the same e.g. if the temperature after 30 seconds is not -10 then show a warning "microscope will power off in 10 seconds.." and shutdown.
Do you agree?

**Comment by zmikulski** [comment]
Yes, I think this makes total sense. I would add a sentence or two of the explanation and steps to resolve the issue (eg. link to a video with Mike filling the tubes). If you can automatically ping your support team to let them know that something is wrong it would be awesome. Prompt response from them would be very valuable in the eyes of the customer. 

Did I mention that we managed to fry 2 Orca cameras on a fully enclosed, badly designed commercial system that was supposed to run 24/7 but didn't monitor the temperature and didn't include external cooling? ;)

**Comment by trygub** [comment]
After a short trial run we've decided to phase out ONI's support repo on Github in favour of a more flexible issue tracking system.
New system is not live yet and while we make this transition let us continue this discussion by email (support@oxfordni.com).
Thank you for your valuable feedback.
We will endeavour to resolve your issue as quickly as possible.
